src/pmi/pmi.cs

```diff
@@ -1270,28 +1275,30 @@ private static int Usage()
         }
 
         Console.WriteLine(
-            "Usage:\r\n"
-            + "\r\n"
-            + "  " + exeName + " Count PATH_TO_ASSEMBLY\r\n"
-            + "      Count the number of types and methods in an assembly.\r\n"
-            + "\r\n"
-            + "  " + exeName + " PrepOne PATH_TO_ASSEMBLY INDEX_OF_TARGET_METHOD\r\n"
-            + "      JIT a single method, specified by a method number.\r\n"
-            + "\r\n"
-            + "  " + exeName + " PrepAll PATH_TO_ASSEMBLY [INDEX_OF_FIRST_METHOD_TO_PROCESS]\r\n"
-            + "      JIT all the methods in an assembly. If INDEX_OF_FIRST_METHOD_TO_PROCESS is specified, it is the first\r\n"
-            + "      method compiled, followed by all subsequent methods.\r\n"
-            + "\r\n"
-            + "  " + exeName + " DriveAll PATH_TO_ASSEMBLY\r\n"
-            + "      The same as PrepAll, but is more robust. While PrepAll will stop at the first JIT assert, DriveAll will\r\n"
-            + "      continue by skipping that method.\r\n"
-            + "\r\n"
-            + "Environment variable PMIPATH is a semicolon-separated list of paths used to find dependent assemblies.\r\n"
-            + "\r\n"
-            + "Optional suffixes will change behavior:\r\n"
-            + "   -CCtors will jit and run cctors before jitting other methods\r\n"
-            + "   -Quiet will suppress in-progress messages for type and method exploration\r\n"
-            + "   -Time will always show elapsed times, even in -Quiet mode"
+            "Usage:\n"
```

Comment:
I presume Console.WriteLine does embedded `\n` => `\r\n` translation on Windows? (but not on Unix)

Reply:
I can't find anything official, but it works fine; I've never actually used explicit `\r\n`.

Reply:
I presume explicit `\r\n` isn't great on Unix.